src/styles/transitions/_text-page-load.scss

```diff
@@ -0,0 +1,69 @@
+.text-page-load {
```

Comment:
The top of the page looks a little cramped to me. Try adding a little padding on top of the `text-page-load` container.